src/devices/Rtc/Rtc.csproj

```diff
@@ -10,7 +10,7 @@
       <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
     </ProjectReference>
     <ProjectReference Include="..\Units\Units.csproj" />
-    <Compile Include="..\Common\Iot\Device\Common\NumberHelper.cs" />
+    <ProjectReference Include="..\Common\Iot\Device\Common\CommonHelpers.csproj" />
```

Comment:
should this be in the sample?

Reply:
No, the Rtc was actually using NumberHelper but now I made it use the ProjectReference instead.

Reply:
right